packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,942 @@
+        <Option value="test-one">{'https://test.twilio.com/welcome/voice/long-test/too-long/text-continuing'}</Option>
+        <Option value="test-two">{'https://test.twilio.com/welcome/voice/long-test/too-long/text-continuing/still-continuing'}</Option>
+      </Select>
+      </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Keep help text and error text concise and simple."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="help_text_length_do">HTTP method</FormLabel>
+      <Select id="help_text_length_do" name="help_text_length_do" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+      <FormHelpText variant="default" id="help_text_length_do">HTTP action for voice webhook.</FormHelpText>
+    </Box>
+  </Do>
+  <Dont center title="Don't" body="Don’t use more than 2 sentences in help text or error text.">
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="help_text_length_dont">HTTP method</FormLabel>
+      <Select id="help_text_length_dont" name="help_text_length_dont" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+      <FormHelpText variant="default" id="help_text_length_dont">HTTP action for voice webook. This will be used to manage an incoming call. Please also configure your fallback url and method in case the primary handler fails.</FormHelpText>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do center title="Do" body="Include a visible label on every form field.">
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="use_label_do">HTTP method</FormLabel>
+      <Select id="use_label_do" name="use_label_do" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont center title="Don't" body="Don’t use prefix/suffix text as a replacement for a label.">
+    <Box width="100%" padding="space60">
+      <Select id="use_label_dont" name="use_label_do" insertBefore={<span>HTTP</span>} onChange="" >
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Use a disabled form field to show users that they can’t interact with the field, but that it could potentially be enabled through another UI element."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="disabled_do">HTTP method</FormLabel>
+      <Select disabled id="disabled_do" name="disabled_do" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
```

Comment:
Hrmmm maybe this example could have help text explaining why the field is disabled to distinguish it from the Don't example? Maybe "Please sign in again to change your HTTP method".

Or this could be a read-only input instead.

Reply:
Yeah, that's a great idea, will make that change 😄